src/appleseed/renderer/modeling/input/sourceshadingpoint.cpp

```diff
@@ -0,0 +1,14 @@
+// Interface header.
+#include "sourceshadingpoint.h"
+
+namespace renderer
+{
+    SourceShadingPoint::SourceShadingPoint(const foundation::Vector2f uv)
```

Comment:
We don't indent the content of top-level namespaces (since that amounts to indenting the whole file for no gain).

Reply:
Pass `uv` by const ref.